packages/lib-user/src/components/GroupStats/GroupStats.js

```diff
@@ -55,18 +61,41 @@ function GroupStats({
   // set top projects based on selected date range and all project stats
   let topProjects = []
   const topProjectContributions = allProjectsStats.project_contributions
-    .sort((a, b) => b.count - a.count)
+    ?.sort((a, b) => b.count - a.count)
 
   topProjects = topProjectContributions
-    .map(projectContribution => {
+    ?.map(projectContribution => {
       const projectData = projects?.find(project => project.id === projectContribution.project_id.toString())
       return projectData
     })
     .filter(project => project)
     .slice(0, 6)
 
   return (
-    <Layout>
+    <Layout
+      primaryHeaderItem={
+        <HeaderLink
+          href='/projects'
```

Comment:
This should be hardcoded to `https://www.zooniverse.org/projects` because clicking the link jumps a user over to app-project. The user does not stay within app-root (yet).

Reply:
Yes! Sorry I thought I refactored these accordingly (app-root links relative, PFE, app-project, or app-content-pages should be absolute).